lib/kvdb/ikvdb.c

```diff
@@ -1391,18 +1382,18 @@ ikvdb_open(
     struct kvdb_rparams *params,
     struct ikvdb **      handle)
 {
-    merr_t                 err;
-    struct ikvdb_impl *    self = NULL;
-    atomic_ulong          *tseqnop;
-    u64                    seqno = 0; /* required by unit test */
-    ulong                  mavail;
-    size_t                 sz;
-    int                    i, n;
-    uint32_t               flags;
-    u64                    ingestid, gen = 0, txhorizon = 0;
     struct wal_replay_info rinfo = {0};
-    struct mpool_rparams   mparams;
-    struct kvdb_meta       meta;
+    struct mpool_rparams mparams;
+    struct kvdb_meta meta;
+    struct ikvdb_impl *self = NULL;
+    atomic_ulong *tseqnop;
```

Comment:
Should this be an atomic_ullong

Reply:
hmm, yes, if we want our code to be portable and run on Windows some day :-)

Reply:
Even better if you use one of the fixed width builtin atomic types